docs/SNAPTEL.md

```diff
@@ -186,16 +187,18 @@ and then:
 3. load a publishing plugin
 4. list the plugins
 5. start a task with a task manifest
-6. start a task with a workflow manifest
-7. list the tasks
-8. unload the plugins
+6. make a single run task
+7. start a task with a workflow manifest
```

Comment:
Hmm, it seems that no special task is created for single run task - how about removing 6. and in 7. saying: "start a single run task with a workflow manifest" ?